prow/manifests/config_configmap.yaml

```diff
@@ -0,0 +1,77 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  namespace: default
+  name: config
+data:
+  config.yaml: |
```

Comment:
All the content will be in git twice if this is checked in.  Do you have an idea for how to ensure the files in git stay in sync?  Or you could drop this since it's easy to create file the yaml file.  The same applies to the plugins.yaml configmap.

Reply:
Sorry this file should not be there. config.yaml and plugins.yaml is enough.

Reply:
> Sorry this file should not be there. config.yaml and plugins.yaml is enough.

EDIT:
This file is the initial ConfigMap  that we need to create when we are re-deploying all the prow manifests from scratch. We won't be modifying this file in future, and instead modify `/prow/config/config.yaml` one if we want to remove/add more jobs or do any sort of changes. I can remove its content so that people don't get confused but we still create an empty ConfigMap, but it is good to have it, in case we need to re-deploy the prow from scratch in the future. I can make it clear in the doc also that we need to work with `/prow/config/config.yaml ` and not `prow/manifests/config_configmap.yaml`

Reply:
@russellb I also disabled/commented out auto-bump for now. I will need to check if we are going to use it or now. In the worse case, we can do the manual update of images.